.gitignore

```diff
@@ -24,6 +24,7 @@ nosetests.xml
 eslint.xml
 jest.xml
 local_testing*
+output*
```

Comment:
pls remove - its better to put local testing output in /tmp or something like this.

Reply:
okay